app/views/events/_schedule_league.html.erb

```diff
@@ -4,7 +4,9 @@
     <tr class="gameday">
       <th colspan="6">
         <b>
-          <%= t '.gameday' %> <%= match.gameday + 1 %>
+          <%= t 'events.schedule.gameday' %> <%= match.gameday %></b>
+        <b>
+          <%= "- #{ (@event.startdate_for_gameday match.gameday).to_s(:day_and_month) } #{t 'events.schedule.to' } #{(@event.enddate_for_gameday match.gameday).to_s(:day_and_month)}"%>
```

Comment:
I think seperating this in multiple lines would make it more readable.
I would suggest something like:
```
<b>
  - 
  <%= (@event.startdate_for_gameday match.gameday).to_s(:day_an_month) %>
  <%= t 'events.schedule.to' %>
  <%= (@event.enddate_for_gameday match.gameday).to_s(:day_and_month) %>
</b>
```

More statements but no unreadable long line of text concats.

Reply:
Still need to add a space so that it doesn't read like 24.12.to31.12. but 24.12. to 31.12.
but yes splitting is a good idea anyways :)